### PR TITLE
Style front

### PR DESCRIPTION
**Pull request by mikeyrayvon**
uses masonry and skrollr 🥇

### CODE REVIEW

File: css/site.styl [hunk 1]
```diff
@@ -88,6 +88,42 @@ img
...
 
+// HOME
+
+.home-work-item
+  opacity: 0
+  transition: opacity .2s
+
+.home-work-contents
+  position: relative
+  font-size: 0
```
Comment (varffs) [inline css/site.styl]: what is this for?

File: front-page.php [hunk 2]
```diff
@@ -24,37 +24,50 @@
...
 
-        <div id="home-work-holder" class="grid-row">
+        <div id="home-work-holder" class="grid-row padding-top-mid padding-bottom-large js-masonry margin-bottom-large">
 
 <?php
       foreach ($works as $work) {
         if (!empty($work['work_id'])) {
 
-          $work_image = get_the_post_thumbnail($work['work_id'], 'home-work', array('class'=>'home-work-image'));
+          $work_image = get_the_post_thumbnail($work['work_id'], 'full', false, array('data-no-lazysizes'=>true, 'class'=>'home-work-image'));
```
Comment (varffs) [inline front-page.php]: `full` needs to be set to a registered size that makes sense for this design. I guess scale to width that is the max possible with this logic

File: front-page.php [hunk 3]
```diff
@@ -24,37 +24,50 @@
...
-                if (!empty($work_cats)) {
-                  echo '<ul>';
-                  foreach ($work_cats as $cat) {
-                    echo '<li>' . $cat->name . '</li>';
-                  }
-                  echo '</ul>';
-                }
-              ?>
+          <a class="home-work-item grid-item item-s-12 item-m-6 item-l-4" href="<?php echo $work_url; ?>"
+            data-0-bottom-top="transform[sqrt]: translateY(<?php echo -($trans_y); ?>%) translateX(<?php echo -($trans_x); ?>%) scale(.7);"
```
Comment (varffs) [inline front-page.php]: maybe not for this branch but could we put the scale value and maybe the random max mins as site settings?

File: js/main.js [hunk 4]
```diff
@@ -36,6 +45,66 @@ Site = {
...
+    _this.layoutWork();
+    _this.bindImageHover();
+    _this.initSkrollr();
+  },
+
+  initSkrollr: function() {
+    var _this = this;
+
+    _this.skrollr = skrollr.init({
+      forceHeight: false // idk why but this helps
```
Comment (varffs) [inline js/main.js]: is this what fixes the footer issue?

File: js/main.js [hunk 5]
```diff
@@ -36,6 +45,66 @@ Site = {
...
+    var _this = this;
+
+    if ($('.js-masonry').length) {
+      _this.$work = $('.js-masonry').masonry({
+        transitionDuration: 0, // no fucking animations :<
+      });
+
+      // masonry layout callback
+      _this.$work.on( 'layoutComplete',
+        function( event, laidOutItems ) {
```
Comment (varffs) [inline js/main.js]: both vars unused. linting should be complaining I think

**Review by varffs — COMMENTED** [review]
I think the image size should be set in this branch. Otherwise everything else is just clarification

File: css/site.styl (hunk 1 above)
Comment (mikeyrayvon) [reply css/site.styl]: ah it was to kill the whitespace below the iamge but i shoudl just be setting it to block level

File: front-page.php (hunk 3 above)
Comment (mikeyrayvon) [reply front-page.php]: yea i'll make a card

File: js/main.js (hunk 4 above)
Comment (mikeyrayvon) [reply js/main.js]: kind of yea.  i'll fix that comment

File: js/main.js (hunk 5 above)
Comment (mikeyrayvon) [reply js/main.js]: true

File: front-page.php (hunk 2 above)
Comment (mikeyrayvon) [reply front-page.php]: ok

File: front-page.php (hunk 3 above)
Comment (mikeyrayvon) [reply front-page.php]: actually i'll just do it now fi

**Pull request merged**